app/javascript/hooks/queries/recordings/useRecordings.jsx

```diff
@@ -1,8 +1,11 @@
 import { useQuery } from 'react-query';
-import axios, { ENDPOINTS } from '../../../helpers/Axios';
+import axios from 'axios';
 
-export default function useRecordings() {
-  return useQuery('getRecordings', () => axios.get(ENDPOINTS.recordings, {
+export default function useRecordings(input) {
+  return useQuery(['getRecordings', input], () => axios.get('/api/v1/recordings.json', {
+    params: {
+      q: input,
```

Comment:
Naming this param `q` is weird lol

Reply:
Would "query" be better?

Reply:
`search` would be best

Reply:
"search" might be better

Reply:
Question: What is making the query "re-fire" each time a letter is typed? Is it when the onChange changes the value of the query param, it automatically re-fires the query since the param changed?

Reply:
It's the input used as a query-key.
Query will fire every time the input, the key, changes.

Reply:
useQuery(['getRecordings', input], ...)

Reply:
Oh okay cool I see it now :+1: